lib/src/Base/Stat/openturns/CovarianceModel.hxx

```diff
@@ -72,19 +72,13 @@ public:
   virtual Scalar computeStandardRepresentative(const Scalar & s,
       const Scalar & t) const;
 
-  virtual Scalar computeStandardRepresentative(const Point & tau) const;
-
-  virtual Scalar computeStandardRepresentative(const Scalar & tau) const;
```

Comment:
This should be reflected in the doc:
http://openturns.github.io/openturns/master/user_manual/_generated/openturns.CovarianceModel.html?highlight=covariancemodel#openturns.CovarianceModel.computeStandardRepresentative